README.md

```diff
@@ -50,6 +50,10 @@ pub fn main() void {
     // Or...golang magic date specifiers. Format strings are not required to be comptime
     try dt.gofmt(anywriter, "2006-01-02 15:04:05 MST");
 
+    // Format using strftime straight to buffer.
+    var buf: [128]u8 = undefined;
+    const formatted_date_string = try dt.bufStrftime(buf, "%Y-%m-%d %H:%M:%S %Z");
+
```

Comment:
I'm not convinced...I don't think expanding the API of zeit is worth saving two lines of code at the callsite:
```zig
    var buf: 128[u8] = undefined;
    var fbs = std.io.fixedBufferStream(&buf);
    try dt.strftime(fbs.writer().any(), "%Y-%m-%d %H:%M:%S %Z");
    const formatted_date_string = fbs.getWritten();
```

Reply:
Yeah that's fair! I can definitely respect that :)

As an alternative, maybe there should be a `fixedBufferStream` example in the readme? It would've made my life easier as someone new to zig 🤔 

Knowing more about what `anywriter` means now I can absolutely see how that would be enough for anyone familiar with zig, so adding a bigger example to the readme would probably only benefit people new to zig.

The example I'm thinking of is exactly what you've outlined in this thread, so it wouldn't be a big addition.